src/mame/drivers/namcos2.cpp

```diff
@@ -813,47 +813,40 @@ void namcos2_state::common_suzuka8h_am(address_map &map)
 	map(0x840000, 0x840001).nopr();
 	map(0x900000, 0x900007).rw(m_c355spr, FUNC(namco_c355spr_device::position_r), FUNC(namco_c355spr_device::position_w));
 	map(0xa00000, 0xa1ffff).rw(m_c45_road, FUNC(namco_c45_road_device::read), FUNC(namco_c45_road_device::write));
+	map(0xc00000, 0xc0ffff).noprw(); // no ROZ hardware implemented in PCB
+	map(0xd00000, 0xd0001f).noprw(); // ^^
```

Comment:
I'd rather this be in the `suzuka8h` map rather than having the note in this common function that's also used by systems mapping the ROZ hardware over the top.